src/codegen/cfg.rs

```diff
@@ -503,13 +504,23 @@ impl ControlFlowGraph {
                 self.expr_to_string(contract, ns, l),
                 self.expr_to_string(contract, ns, r)
             ),
-            Expression::Divide(_, _, l, r) => format!(
-                "({} / {})",
+            Expression::SignedDivide(_, _, l, r) => format!(
+                "signed({} / {})",
```

Comment:
This reads like "cast the result of divide to unsigned". That is not very clear. How about "(unsigned divide {} / {})"?